packages/gatsby-image/src/withIEPolyfill/index.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Image from "../index"
+
+// Dynamically load polyfill for object-fit/object-position if needed (e.g. in IE)
+const testImg = document.createElement(`img`)
+if (
+  typeof testImg.style.objectFit === `undefined` ||
+  typeof testImg.style.objectPosition === `undefined`
+) {
+  import(`object-fit-images`).then(({ default: ObjectFitImages }) =>
+    ObjectFitImages()
```

Comment:
I can read `This will fix all the images on the page and also all the images added later (auto mode).` on the github page but I can't see how they actually achieve this. It looks like they listen for onload but nothing after that.

I would go for componentDidMount and if possible createRef so we can only trigger it on the current image so we don't do to much work. If you need an example of what I'm thinking feel free to ask.

Reply:
Sounds good! I assume the ref will need to be attached to the actual `<img />` element rendered by the `<Image />` component?

Since the `<img />` element isn't exposed in this wrapper component, any tips on how to drill the ref down to it?

Reply:
```js
const imageRef = React.createRef();

componentDidMount() {
ObjectFitImages(imageRef.current.querySelector('img'))
}
```

something like this might work?

Reply:
Of course! Silly me. 🤦‍♂️ On it.